src/ServiceControl.Transports.RabbitMQ/ConnectionConfiguration.cs

```diff
@@ -18,7 +18,7 @@ class ConnectionConfiguration
         const string defaultVirtualHost = "/";
         const string defaultUserName = "guest";
         const string defaultPassword = "guest";
-        const ushort defaultRequestedHeartbeat = 5;
+        const ushort defaultRequestedHeartbeat = 60;
```

Comment:
Why did we change this?

Reply:
This was changed in https://github.com/Particular/NServiceBus.RabbitMQ/pull/608 to align our default value with the client's actual default. Five seconds was way too short, and we should never have been setting it that low.